var/spack/repos/builtin/packages/py-matplotlibscalebar/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMatplotlibscalebar(PythonPackage):
```

Comment:
```suggestion
class PyMatplotlibScalebar(PythonPackage):
```
Can you also rename the directory to `py-matplotlib-scalebar`? There's a missing hyphen.